tests/Unit/AntispamBeeTest.php

```diff
@@ -68,7 +68,7 @@ public function test_gets_ip_address() {
 		$_SERVER['REMOTE_ADDR']          = '192.0.2.1';
 		$_SERVER['HTTP_X_FORWARDED_FOR'] = '192.0.2.2, 10.0.0.10';
 		$_SERVER['HTTP_X_REAL_IP']       = 'bogus';
-		$_SERVER['REQUEST_URI']          = 'https://domain.com/wp-comments-post.php';
+		$_SERVER['SERVER_NAME']          = 'https://domain.com/wp-comments-post.php';
```

Comment:
Guess you wanted to use `SCRIPT_NAME` here?

This car typically does not contain the full URL, so a more realistic value would be
```php
$_SERVER['SCRIPT_NAME']          = '/wp-comments-post.php';
```

Reply:
Right!

Reply:
But the `REQUEST_URI` would often be the same, so the code was wrong before 😅 

@stklcode do you know why the tests are broken?

Reply:
I think I've found the issue 🙈 The error message wasn't really of big help 😞

Reply:
@stklcode any idea on why the Behat tests are failing?

Reply:
Unfortunately no. I'm not really familiar with _Behat_ and the bootstrap script is apparently only just running (behat.sh even throws some errors that are ignored) in GH actions, so it's little tricky to debug.

But the failure seems unrelated to your changes. I reverted everything temporarily and CI still fails: https://github.com/pluginkollektiv/antispam-bee/actions/runs/9066345933/job/24909046007?pr=589

Reply:
Likely a currently broken language API.

Reply:
So you would say we can merge this PR?

Reply:
Yes. Changes are reasonable and look good.

... whenever GH actions finally reports the status, system is degraded right now.